src/libc++.vcxproj

```diff
@@ -20,19 +20,31 @@
     <ClCompile Include="chrono.cpp" />
     <ClCompile Include="condition_variable.cpp" />
     <ClCompile Include="condition_variable_destructor.cpp" />
-    <ClCompile Include="debug.cpp" />
     <ClCompile Include="exception.cpp" />
+    <ClCompile Include="filesystem\directory_entry.cpp" />
+    <ClCompile Include="filesystem\directory_iterator.cpp" />
+    <ClCompile Include="filesystem\filesystem_clock.cpp" />
+    <ClCompile Include="filesystem\filesystem_error.cpp" />
+    <ClCompile Include="filesystem\operations.cpp" />
+    <ClCompile Include="filesystem\path.cpp" />
```

Comment:
@solemnwarning I recall from our conversation that you had problems with `std::filesystem` and disabled it entirely.
If that is still the case, we can probably skip compiling everything from the `filesystem` subdirectory, just like I did for libc++ 11 and VS2019.

Or is `std::filesystem` supported now?

Reply:
I had to re-enable it for one of the libraries I'm using in rehex, however the compile errors went away like magic, it might depend exactly what `std::filesystem` functionality is being used in the application.

I had another issue where libc++'s `std::chrono` was linking against `GetSystemTimePreciseAsFileTime()`, which I made a note of to investigate and fix later (probably with another wrapper in EnlyzeWinCompatLib), but when I went to do that, it had resolved itself.

I think perhaps I was compiling with the wrong _WIN32_WINNT at some point, and exactly how much works on what Windows version is going to depend on exactly what C++ functionality the final application is using.

Reply:
Just as an FYI, I threw this together to verify libc++ in general still works in NT4: https://github.com/solemnwarning/nt4-exe-test

It uses a bit of STL, and threads/atomics/exceptions, which is obviously leaving large swathes of libc++ untested... it would be nice to get libc++'s test suite compiled and running on an NT4 target, but thats probably a huge amount of work neither of us really want to get into right now :D